--- sw.js	2023-01-18 06:26:05.000000000 -0500
+++ ../../html/sw.js	2023-07-28 08:29:50.904558017 -0400
@@ -1,6 +1,6 @@
 const cacheID = "v0";
 const CacheItems = [
-    "index.html",   // Special page: Loads from network
+    "phone.shtml",   // Special page: Loads from network
     "offline.html",   // Special page: Save to cache, but return only when offline
 
     "/favicon.ico",
@@ -77,7 +77,7 @@
 });
 
 self.addEventListener("fetch", function(event){
-    if(event.request.url.endsWith("index.html")){
+    if(event.request.url.endsWith("phone.shtml")){
         console.log("Special Home Page handling...", event.request.url);
         event.respondWith(loadHomePage(event.request));
     }
